### PR TITLE
Adding types

### PR DESCRIPTION
**Pull request by jdtoombs**
Just cleaning things up a little

### CODE REVIEW

File: frontend/src/components/common/form/AutoCompleteText.tsx [hunk 1]
```diff
@@ -5,32 +5,33 @@ import { Form } from 'react-bootstrap';
...
-  const [suggestions, setSuggestions] = useState<any>([]);
-  const [text, setText] = useState<any>('');
+  const [suggestions, setSuggestions] = useState<SelectOption[]>([]);
+  const [text, setText] = useState<string>('');
 
-  const onTextChanged = (e: any) => {
+  const onTextChanged = (e: React.ChangeEvent<HTMLInputElement>) => {
     const val = e.target.value;
-    let dynamicSuggestions: any[] = [];
+    let dynamicSuggestions: SelectOption[] = [];
```
Comment (Fosol) [inline frontend/src/components/common/form/AutoCompleteText.tsx:24]: Is `SelectOption` an object or an interface?  If it's an interface it should be renamed.

File: frontend/src/components/common/form/AutoCompleteText.tsx [hunk 2]
```diff
@@ -5,32 +5,33 @@ import { Form } from 'react-bootstrap';
...
-  const [suggestions, setSuggestions] = useState<any>([]);
-  const [text, setText] = useState<any>('');
+  const [suggestions, setSuggestions] = useState<SelectOptions>([]);
+  const [text, setText] = useState<string>('');
 
-  const onTextChanged = (e: any) => {
+  const onTextChanged = (e: React.ChangeEvent<HTMLInputElement>) => {
     const val = e.target.value;
-    let dynamicSuggestions: any[] = [];
+    let dynamicSuggestions: SelectOptions = [];
```
Comment (Fosol) [inline frontend/src/components/common/form/AutoCompleteText.tsx:24]: Is SelectOption an object or an interface? If it's an interface it should be renamed.

Comment (asanchezr) [reply frontend/src/components/common/form/AutoCompleteText.tsx:24]: I don't know if it is an interface. I'm okay if we keep renaming the interfaces to `IWhatever` but that's not common in typescript codebases

Comment (jdtoombs) [reply frontend/src/components/common/form/AutoCompleteText.tsx:24]: I'll merge this for now just so we can mark the autocomplete as done, but I'll adjust the name to ISelectOption / ISelectOptions now as well.

**Pull request merged**